### DISCUSSION TITLE
MEVO + Wiki Needs update

### DISCUSSION THREAD

**Discussion by RocknRollGolf** [springbok/MLM2PRO-GSPro-Connector#92] [General]
Select 'Devices' from the toolbar to open the Devices window.
Press the 'New' button to add a new device.
Enter the following values:
Device Name: 'Mevo+' or a device name of your choosing. **_### Dont use the  +  Sign_** 
Mirror App Window Name: 'FS GOLF PC 2.0' It's important to get this right otherwise the Connector won't be able to locate the correct window when taking a screenshot.   **_**If you are using Updated FS Golf PC , Dont place 2.0 at the end **_**
Mirror App Path(Optional): The Connector can automatically start the Mevo FS Golf application for you if it's not already running when selecting a device. Use the Browse button and locate the correct file to start

**Comment by springbok** [comment]
Thanks for the updates I will add them in when I get back next week.
